service/src/main/java/bio/terra/cbas/dao/RunDao.java

```diff
@@ -128,14 +128,6 @@ public int updateLastPolledTimestamp(UUID runID) {
                 DateUtils.currentTimeInUTC())));
   }
 
-  public int updateErrorMessage(UUID runId, String updatedErrorMessage) {
```

Comment:
Was this method not used?

Reply:
Yeah, this is just "while I'm there" tidying up